### PR TITLE
feat(structure): change gem structure from bundler to rails plugin

### PR DESCRIPTION
**Pull request by sjoulbak**
I ran `rails plugin new lti-rails --mountable` outside of the gems folder in order to generate the necessary files. I kept here and there some of the generated files/information of bundler.

As a chore I've incremented the supported ruby version to 3.0, because < 3.0 is EOL.

### CODE REVIEW

**Review by dennisdevulder — APPROVED** [review]

File: app/helpers/lti/rails/application_helper.rb
Comment (dennisdevulder) [inline app/helpers/lti/rails/application_helper.rb:1]: Should be removed

File: app/mailers/lti/rails/application_mailer.rb
Comment (dennisdevulder) [inline app/mailers/lti/rails/application_mailer.rb:1]: Should be removed

File: test/dummy/app/channels/application_cable/channel.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+module ApplicationCable
+  class Channel < ActionCable::Channel::Base
```
Comment (Robinoudev) [inline test/dummy/app/channels/application_cable/channel.rb:4]: and this

File: test/dummy/app/channels/application_cable/connection.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+module ApplicationCable
```
Comment (Robinoudev) [inline test/dummy/app/channels/application_cable/connection.rb:3]: and this?

File: test/dummy/app/helpers/application_helper.rb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+module ApplicationHelper
```
Comment (Robinoudev) [inline test/dummy/app/helpers/application_helper.rb:3]: and tihs

File: test/dummy/app/mailers/application_mailer.rb [hunk 4]
```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class ApplicationMailer < ActionMailer::Base
```
Comment (Robinoudev) [inline test/dummy/app/mailers/application_mailer.rb:3]: and this

**Review by Robinoudev — APPROVED** [review]
LGTM

**Pull request merged**